src/main/webapp/script.js

```diff
@@ -1,13 +1,13 @@
 // Copyright 2020 Google LLC
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
+// Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
Don't change the license text. Same comment on line 10.

Reply:
sorry for that, I was change " to ' in the code to keep them consistent and I just did ctrl+h